pkg/virt-controller/watch/export/export.go

```diff
@@ -777,19 +822,58 @@ func (ctrl *VMExportController) updateVMExportPvcStatus(vmExport *exportv1.Virtu
 			}
 		}
 	}
-	if !equality.Semantic.DeepEqual(vmExport, vmExportCopy) {
-		if _, err := ctrl.Client.VirtualMachineExport(vmExportCopy.Namespace).Update(context.Background(), vmExportCopy, metav1.UpdateOptions{}); err != nil {
-			return retry, err
+	return internalLink, nil
+}
+
+func (ctrl *VMExportController) getExternalLinks(pvcs []*corev1.PersistentVolumeClaim, exporterPod *corev1.Pod, export *exportv1.VirtualMachineExport) (*exportv1.VirtualMachineExportLink, error) {
```

Comment:
looks like a lot of shared code with getInternalLinks().  Any opportunity for sharing?  Maybe have hostname and path prefix parameters? Or create functions for each of the different contenttypes?